v2.4/getting-started/kubernetes/troubleshooting.md

```diff
@@ -54,3 +54,13 @@ However, if you do need to assign a particular address to a Pod, Calico provides
 - You can request an IP using the `cni.projectcalico.org/ipAddrsNoIpam` annotation. Note that this annotation bypasses the configured IPAM plugin, and thus in most cases it is recommended to use the above annotation. 
 
 See the [Requesting a Specific IP address]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration#requesting-a-specific-ip-address) section in the CNI plugin reference documentation for more details.
+
+#### How can I restrict external access to exposed services (i.e. access to a NodePort)?
```

Comment:
It is a little odd to include an `i.e.` in a heading. In general, I don't think it is a good idea. I would suggest revising this heading to remove the `i.e.` but in the future, please know that there should always be a comma after `i.e.` and `e.g.`. I would suggest changing this to "How can I restrict external access to exposed services like NodePorts?" or "How can I restrict external access to exposed services such as NodePorts?"